docs/_includes/Statements.md

```diff
@@ -119,44 +162,45 @@ CallStmt_ =
     [ Lhs { , Lhs } ":=" ] Lhs ";"
 ````
 
-The update statement server three logical purposes.
+The update statement serves several logical purposes.
 
-First, if it matches the grammar of a ``CallStmt_`` in which the right-most ``Lhs``
-references a method and arguments to pass to it, then the ``UpdateStmt`` is a
-method call or a new allocation with an initializing method.
-These are the only ways in which a method call is allowed in an update statement.
-In particular, the result of one method call is not allowed to be used
-as an argument of another method call (as if it had been an expression).
 
-The form
+1) The form
 
 ```
 Lhs {Attribute} ";"
 ```
 is assumed to be a call to a method with no out-parameters.
 
-The form
+2) The form
 
 ```
     [ Lhs { , Lhs } ":=" ] Lhs ";"
```

Comment:
Since this describes a different form, I suggest dropping the square brackets on L178. (The form where the square brackets do not take the optional stuff is covered in Form 1.)